addons/sale_stock/migrations/11.0.1.0/pre-migration.py

```diff
@@ -59,8 +60,8 @@ def update_stock_move_field_from_procurement_order(env):
     )
 
 
-@openupgrade.migrate()
+@openupgrade.migrate(use_env=True)
```

Comment:
There's no need of adding this argument. It's `True` by default. Not blocking any way.